scripts/picard

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
+from picard.tagger import main; main('/usr/share/locale', True)
```

Comment:
I'm sorry, while building the ui, I once mistakenly pressed enter at 'python setup.py build' instead of ..build_ui. This has been generated as a by-product of this. I have removed the file and updated the PR.